### DISCUSSION TITLE
Sensor polling frequency

### DISCUSSION THREAD

**Discussion by Timohasan** [Rem0o/FanControl.Releases#2825] [General]
How often does the SSD temperature sensor poll? Once a minute?

**Comment by Rem0o** [comment]
FanControl polls everything at 1hz.

**Reply by Timohasan** [reply]
Perhaps I expressed myself incorrectly.
I would like to know at what interval the added SSD temperature icon in the tray is updated?
P.S. Sorry for my bad English, it's not my native language.

**Reply by Rem0o** [reply]
Same thing 1hz

**Reply by Timohasan** [reply]
As far as I could see, the number on the tray icon is updated once a minute. Right?

**Reply by Rem0o** [reply]
Once a second.

**Reply by Timohasan** [reply]
It's strange, my SSD readings change every minute in the tray. Apparently I'm doing something wrong :)

**Reply by Rem0o** [reply]
Found the culprit. LHM caps it at once a minute for storage devices:
https://github.com/LibreHardwareMonitor/LibreHardwareMonitor/blob/master/LibreHardwareMonitorLib/Hardware/Storage/AbstractStorage.cs#L23

**Reply by Timohasan** [reply]
Is it possible to change or remove this restriction? Some easy way? :)

**Reply by Rem0o** [reply]
That code is in LHM.  There is a reason this was put in, as I suppose SMART health check is costly and this is how the data gets pulled in.

**Reply by Timohasan** [reply]
Ок. Thank you for helping me understand what the problem was.
